### DISCUSSION TITLE
Profe pa que veas Dr stone V:

### DISCUSSION THREAD

**Discussion by Darksea1220** [sebasvelasco353/FDP-2022_01#190] [Ideas]
[https://www3.animeflv.net/ver/dr-stone-1](url)
Allí está el enlace, nomás procurá cerrar las ventanas emergentes y tener bien el antivirus

**Comment by DParuma** [comment]
Momento sad
Assesination classrom
Shigatsu wa Kimi no
Clannad
Anohana
Golden time
Angel beats
Death note
Vivy
Charlotte

pelicula
Quiero comerme tu pancreas

**Reply by Darksea1220** [reply]
si quieres algo entretenido con un poquito de relleno y un poquito de antojo :u [https://www3.animeflv.net/anime/tsugumomo](url)
